### PR TITLE
Clarify travel reimbursement

### PR DESCRIPTION
**Pull request by choldgraf**
These add a few travel reimbursement guidelines so that we can disambiguate any confusion or questions ahead of time! It tries to be relatively lightweight but hopefully helpful. Comments or suggestions welcome.

### CODE REVIEW

File: administration/reimburse.md [hunk 1]
```diff
@@ -26,10 +26,27 @@ Currently, these are the expenses that we regularly reimburse:
...
+(reimburse:travel)=
+### Travel reimbursements
+
+Below are a few guidelines for travel reimbursements with 2i2c:
+
+- 2i2c will cover the cost of travel to and from the event.
+- We'll cover the cost of lodging for the day before or after.
+- For international travel, we can cover an extra day before to give more time to adjust.
+- It is fine if you are arriving / leaving from two different locations (e.g., if you intend on making a side-trip after the event).
+  In this case, 2i2c will only cover the trip to / from the vent, not the third leg.
```
Comment (pnasrat) [inline administration/reimburse.md:43]: typo vent -> event

Comment (choldgraf) [inline administration/reimburse.md:43]:
```suggestion
  In this case, 2i2c will only cover the trip to / from the event, not the third leg.
```

**Review by damianavila — APPROVED** [review]
These are sensible guidelines, IMHO.

**Comment by choldgraf** [comment]
Folks, please just merge these if you think they are reasonable and they don't have any objections! We don't need dangling PRs with approvals but no objections sitting around for too long :-)

**Pull request merged**